portfolio/src/main/webapp/returnMessage.js

```diff
@@ -0,0 +1,8 @@
+async function returnFacts() {
+  const responseFromServer = await fetch('/MyServlet');
+  const textFromResponse = await responseFromServer.json();
+  const pickFact = [textFromResponse["Fact 1"], textFromResponse["Fact 2"], textFromResponse["Fact 3"]];
```

Comment:
String concatenation is fairly simple in JavaScript, meaning you can actually simplify the choice to something like:

```javascript
const index = Math.floor(Math.random() * 3) + 1;
const fact = textFromResponse[`Fact ${index}`];
```

Additionally, `textFromResponse` is a misnomer, since it's not actually text, but rather the parsed JSON object.